adback-solution-to-adblock/class-ad-back.php

```diff
@@ -27,7 +27,7 @@ public function getContents($url)
         }
     }
 
-    public function postContents($url, $fields, $header = array())
+    public static function postContents($url, $fields, $header = array())
```

Comment:
Or, you can instantiate a new Ad_Back_Generic object (this object have no dependencies) in activator :)